src/main/java/core/basesyntax/model/zoo/Animal.java

```diff
@@ -1,6 +1,18 @@
 package core.basesyntax.model.zoo;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
```

Comment:
please add `@DiscriminatorColumn` and `@DiscriminatorValue` annotations where needed